doc/nrf/libraries/caf/ble_smp.rst

```diff
@@ -47,22 +47,20 @@ After the previously submitted event is processed, the module submits a subseque
 The application user must not perform more than one firmware upgrade at a time.
 The modification of the data by multiple application modules can result in a broken image that is going to be rejected by the bootloader.
 
-You can perform the DFU using, for example, the `nRF Connect for Mobile`_ application.
-The :guilabel:`DFU` button appears in the tab of the connected Bluetooth device that supports the image transfer over the Simple Management Protocol (SMP).
-After pressing the button, you can select the :file:`*.bin` file used for the firmware update.
+After building your application in configuration with the |smp| enabled, the :file:`dfu_application.zip` archive is generated in the build directory.
```

Comment:
```suggestion
After building your application with the |smp| enabled, the :file:`dfu_application.zip` archive is generated in the build directory.
```
Removing this here as well - please let me know in case this part is crucial